db/migrate/20170725120715_add_gender_to_user.rb

```diff
@@ -0,0 +1,17 @@
+class AddGenderToUser < ActiveRecord::Migration[5.0]
+  def up
+    execute <<-SQL
```

Comment:
i find only this
gender like a integer in DB
and in model have Array with name of Gender ["male", "female"]
and when create User i will puts gender: 1 || 2
but i don`t now it is good idea